modules/microshift-configuring-ovn.adoc

```diff
@@ -52,7 +52,7 @@ To customize your configuration, use the following table that lists the valid va
 
 |mtu
 |uint32
-|1400
+|auto
```

Comment:
I based this change off of the chart in the MicorShift repo docs, I'm not sure if its correct

Reply:
Gotcha! Thank ya!